languagetool-language-modules/de/src/main/resources/org/languagetool/rules/de/grammar.xml

```diff
@@ -61343,12 +61343,13 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
                     <token>,</token>
                 </pattern>
                 <message>'Nachdem' drückt standardsprachlich Vorzeitigkeit aus und sollte daher nicht mit dem Präteritum verwendet werden. Verwenden Sie das Perfekt (Präsens im Hauptsatz) oder Plusquamperfekt (Präteritum im Hauptsatz) oder 'als' zum Ausdrücken von Gleichzeitigkeit.</message>
-                <suggestion suppress_misspelled="yes"><match no="2" postag="VER:PA2.*" postag_regexp="yes"/> war</suggestion>
-                <suggestion suppress_misspelled="yes"><match no="2" postag="VER:PA2.*" postag_regexp="yes"/> hatte</suggestion>
+                <suggestion suppress_misspelled="yes"><match no="2" postag="VER:PA2.*" postag_regexp="yes"/> <match no="2" postag="VER:([123].*)PRT.*" postag_replace="VER:AUX:$1PRT" postag_regexp="yes">sein</match></suggestion>
+                <suggestion suppress_misspelled="yes"><match no="2" postag="VER:PA2.*" postag_regexp="yes"/> <match no="2" postag="VER:([123].*)PRT.*" postag_replace="VER:AUX:$1PRT" postag_regexp="yes">haben</match></suggestion>
```

Comment:
Vlt. noch eine `suggestion` für Perfekt?

Reply:
Hast du ein Beispiel? Die Regeln matchen nur bei 'VER.*PRT'

Reply:
Ein Satz wie "Das Kind darf mit seinen Freunden spielen, nachdem es seine Hausaufgaben machte." wäre im Perfekt auch richtig, oder? Also: "Das Kind darf mit seinen Freunden spielen, nachdem es seine Hausaufgaben gemacht hat."
Das heißt vlt. könnte man hier schauen, dass man die Sätze mit Präsens im Hauptsatz rausfiltert und damit eine eigene Regel macht, die dem User auch das Perfekt vorschlägt. Sonst haben wir die Aufforderung nur in der `message` und der User hat nichts zum Klicken, falls so ein Fall auftritt. Weiß aber nicht wie viel AUfwand das jetzt wäre.

Reply:
Danke! Eigentlich wollte ich nur die falschen/fehlenden Suggestions fixen. Deshalb würde ich Perfekt einfach aufnehmen. 

Nach Präsens / Präteritum prüfen sollte mithilfe des Kommas auch funktionieren. Allerdings würde die Zahl der Subrules auf 16 steigen...